fast/stages/00-bootstrap/README.md

```diff
@@ -177,6 +177,8 @@ Before the first run, the following IAM groups must exist to allow IAM bindings
 - gcp-security-admins
 - gcp-support
 
+![fast-create-groups-v1](https://user-images.githubusercontent.com/94000358/173355158-e6019d48-831b-4f4f-a470-38e2894cd2c8.gif)
```

Comment:
Can you add the image to this repo so it's self-consistent? Maybe in an `.images` folder so it won't pollute listings too much